modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/css.jsp

```diff
@@ -35,19 +35,6 @@ LayoutLookAndFeelDisplayContext layoutLookAndFeelDisplayContext = new LayoutLook
 	/>
 </liferay-frontend:fieldset>
 
-<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPS-166479") %>'>
-	<liferay-frontend:fieldset
```

Comment:
This UI bit about the fieldset, and any other UI subject to the FF existence has to be kept. Removing FF means indeed removing the FF checks, therefore, logic/UI which was being gated under such check is now unconditionally applied.

Reply:
So, here would only be necessary to remove lines 38 and 49? @dsanz @CarlosBrichete